mayhem/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM fuzzers/go-fuzz:1.2.0
```

Comment:
Could you please add a package stage here?

Reply:
What do you mean by this?

Reply:
Something like this for example
https://github.com/bmwtsn098/switchboard/blob/afc763cdc3a1623c5f384fd7ee878a2c3c7ebcdf/mayhem/Dockerfile#L20-L23
Package stages copy the resulting binary artifacts to a clean image (say ubuntu), and install only runtime dependencies (if needed) and nothing else. This helps keep image size down.